src/main/java/pb360/data/entity/id/MessageObjDTO.java

```diff
@@ -0,0 +1,54 @@
+package pb360.data.entity.id;
+
+
+import java.util.Date;
+
+import pb360.data.entity.*;
+
+import pb360.model.MessageObject;
+
+
+
+public final class MessageObjDTO {
+	private String id;
```

Comment:
remove this class, because it's duplicate

Reply:
i got it